test/issue895-local-configuration.sh

```diff
@@ -61,7 +61,7 @@ fi
 rm ../etc/dub/settings.json
 echo "Empty file named ldc2." > ../bin/ldc2
 
-if ! { ${DUB} describe --single issue103-single-file-package.d 2>&1 || true; } | grep -cF "Failed to invoke the compiler $(dirname $CURR_DIR)/bin/ldc2 to determine the build platform"; then
+if ! { ${DUB} describe --single issue103-single-file-package.d 2>&1 || true; } | grep -cF "Failed to execute '$(dirname $CURR_DIR)/bin/ldc2'"; then
```

Comment:
uh do you know why this failed?

Reply:
Yes? I explained it in the commit message.